exercises/custom-set/example.ml

```diff
@@ -1,3 +1,5 @@
+open Core.Std
+
```

Comment:
> It seems best to keep consistent.

I think it would be worth reducing the number of dependencies where possible, even if other exercises do use it.